src/greeter/theme/Main.qml

```diff
@@ -110,6 +110,8 @@ Rectangle {
 
         Row {
             anchors.fill: parent
+            // hide main input for small displays when password change dialog is active
+            visible: !(passwordChange.visible && container.height < 768)
```

Comment:
Shouldn't break point be a param instead of hard coded here and elsewhere?

Reply:
Sure you can put that size value in a param, but I think its not so important at the moment.
If the pull request ever gets accepted - and thats the question - you are free to change it as you like. Someone might also completely redesign the pwd change dialog to fit in the theme, or overlay it, I don't mind.